app/js/components/Workflows.jsx

```diff
@@ -2,24 +2,22 @@ import React from 'react';
 
 import Workflow from './Workflow';
 
-const Workflows = ({ plugin }, { router }) => (
+const Workflows = ({ plugin, openWorkflow }) => (
     <div>
         { plugin.workflows.map(workflow => (
             <Workflow
                 key={workflow.name}
                 flow={workflow}
-                onClick = {() => router.push(`job/${plugin.name}/${workflow.name}`)}
+                onClick = {() => openWorkflow(plugin, workflow)}
             />
         ))}
     </div>
 );
 
 
 Workflows.propTypes = {
-    plugin: React.PropTypes.object
-};
-
-Workflows.contextTypes = {
+    openWorkflow: React.PropTypes.func,
+    plugin: React.PropTypes.object,
     router: React.PropTypes.object
```

Comment:
Back when we were using `router` and `store` from the context, they were required as `contextTypes` or they wouldn't be accessible, but now that we have `withRouter` mapping the router from context to props we shouldn't need it from there, and also properly containerizing components would remove the requirement of needing `store` from the context, so those can probably all go. Removing `propTypes` throws test breaking ESLint errors, but that could be removed through the config if we want.